countries.ts

```diff
@@ -0,0 +1,9532 @@
+import { CountryRaw } from './types';
+
+const countries: { [countryCode: string]: CountryRaw } = {
```

Comment:
I think you should drop the annotation, as it would prevent typescript to actually infer the valid keys (unless you don't want typescript to actually autocomplete country codes)
```suggestion
const countries = {
```

Reply:
done